google-style.yml

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 AllCops:
-  TargetRubyVersion: 2.2
+  TargetRubyVersion: 2.3
```

Comment:
2.3 also adds `Style/FrozenStringLiteralComment`, which I am still not comfortable with for clients that return strings to users.

Reply:
What if we just disable `Style/FrozenStringLiteralComment` for the time being?

Reply:
@quartzmo ^ thoughts on this?

Reply:
I'd be okay disabling `Style/FrozenStringLiteralComment` but I do want to target 2.3.